docs/berkeley-upgrade/migrating-archive-database-to-berkeley.mdx

```diff
@@ -686,6 +543,21 @@ It is worthwhile to perform these last steps as a dry run to make sure all goes
 
 Tips to overcome known challenges.
 
+#### Berkeley migration app is unstable
```

Comment:
maybe say `Known issues` in the heading and list all the issues and the workarounds?